fdbserver/storageserver.actor.cpp

```diff
@@ -709,10 +705,28 @@ struct StorageServer {
 		CounterCollection cc;
 		Counter allQueries, getKeyQueries, getValueQueries, getRangeQueries, finishedQueries, lowPriorityQueries,
 		    rowsQueried, bytesQueried, watchQueries, emptyQueries;
-		Counter bytesInput, bytesDurable, bytesFetched,
-		    mutationBytes; // Like bytesInput but without MVCC accounting
+		Counter
+		    // Bytes of the mutations that have been added to the memory of the storage server. When the data is durable
+		    // and cleared from the memory, we do not exact it but add it to bytesDurable.
+		    bytesInput,
```

Comment:
If these are going to be split across multiple lines, it's probably more stylistically typical of this codebase to make each its own declaration, e.g.:

```
// comment
Counter bytesInput;

//comment
Counter bytesDurable;

...
```

Reply:
Fixed.